src/main/resources/ai/privado/rulevalidator/schema/sinks.json

```diff
@@ -45,6 +45,17 @@
             ],
             "pattern": "^.*$"
           },
+          "filterProperty": {
+            "$id": "#root/sinks/items/filterProperty",
+            "title": "FilterProperty",
+            "type": "string",
+            "default": "method_full_name",
+            "examples": [
+              "code",
+              "method_full_name"
```

Comment:
nitpick: should we use `camelCase` key for `methodFullName`?

Reply:
We have followed underscore-separated values for values like category = third_parties. So I continued with a similar pattern.

Reply:
Ok, makes sense! Lets merge once tests are done 👍